packages/devtools_app/web/debugger_screen.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+
+<!--
+  Copyright 2018 The Chromium Authors. All rights reserved.
+  Use of this source code is governed by a BSD-style license that can be
+  found in the LICENSE file.
+-->
+<html>
+
+<head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script>
+        const GA_DEVTOOLS_PROPERTY = 'UA-26406144-34'; // Dart DevTools GA Property UA.
+
+        function getDevToolsPropertyID() {
+            return GA_DEVTOOLS_PROPERTY;
+        }
+    </script>
+    <!-- The below URI ?id= must match the GA_DEVTOOLS_PROPERTY above. -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-26406144-34"></script>
+    <script type="text/javascript" src="devtools_analytics.js"></script>
+    <!-- End of DevTools Google Analytics -->
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="favicon.png" rel="icon" sizes="64x64">
+    <link href="packages/octicons_css/octicons_4.4.0.css" rel="stylesheet">
+    <link href="packages/polymer_css/flex.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/debugger/debugger.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/inspector/inspector.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/logging/logging.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/performance/performance.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/profiler/profiler.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/info/info.css" rel="stylesheet">
+    <link href="packages/devtools_app/src/timeline/timeline.css" rel="stylesheet">
+
+    <link rel="stylesheet" href="packages/primer_css/primer_12.1.0.css">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="styles_light.css">
+    <style>
+        .container {
+            /* The Flutter app takes care of container sizing now.*/
+            margin-right: 0px;
+            margin-left: 0px;
+        }
+    </style>
+
+    <script src="packages/ansi_up/ansi_up.js"></script>
```

Comment:
add TODO to remove ansi_up.js

Reply:
What is ansi_up used for in the debugger view?

A flutter-specific implementation of ansi will probably end up different from the web implementation and we may not be able to use it.

Reply:
I believe ansi_up is used for debugger console output